pkg/registry/apis/provisioning/webhooks/register.go

```diff
@@ -147,6 +165,29 @@ func (e *WebhookExtra) PostProcessOpenAPI(oas *spec3.OpenAPI) error {
 }
 
 // GetJobWorkers returns job workers from the webhook connector
-func (e *WebhookExtra) GetJobWorkers() []jobs.Worker {
+func (e *WebhookExtraWithImages) GetJobWorkers() []jobs.Worker {
```

Comment:
I think you don't need this and `GetJobWorkers` and `JobsWorkerRetriever` interface. Just split the building process.